checks/raw/fuzzing.go

```diff
@@ -57,6 +58,15 @@ var languageFuzzSpecs = map[clients.LanguageName]languageFuzzConfig{
 		Desc: asPointer(
 			"Go fuzzing intelligently walks through the source code to report failures and find vulnerabilities."),
 	},
+	clients.Cpp: {
+		filePattern: "fuzz_*.cpp",
+		Name:        fuzzerBuiltInCpp,
+		funcPattern: `extern\s+[("C")\s]*[\w\*]+\s+(\w*((?i)fuzz)+\w*)+\s*\([\w* ,]*\)`,
+		URL:         asPointer("https://help.code-intelligence.com/create-a-c-fuzz-test"),
```

Comment:
this is a pretty vendor specific/non-standard way of doing this.

I would just base this on https://github.com/google/fuzzing/blob/master/docs/good-fuzz-target.md#basics instead.

Reply:
Thanks for reviewing my code and providing the google fuzzing target reference!

As you mentioned "this is a pretty vendor-specific/non-standard way", are you referring to the `URL` link I am using for C++ fuzzing in my code, or the `filePattern` and the `funcPattern` which are used to match the files and fuzzing functions?

Reply:
Both: the documentation link points to a vendor specific page, and the pattern should be more selective and only accept LLVMFuzzerTestOneInput.

Reply:
@oliverchang pls see my latest changes to the `URL` fuzz doc, `filePattern` and the `funcPattern`.

Now the `extern "C"` has been excluded from the funcPattern, and the `filePattern` `fuzz_*.c*` can be used to match multiple cpp file extensions such as `.cpp` and `.cc`.

The corresponding unit test cases can be seen [here](https://github.com/ossf/scorecard/pull/2001/files#diff-97593777ce7b8e3b0e3c50d4e8ea916cb608d7c8dae398b083bfb737fcd76d1fR206).

Reply:
resolving this conversation - patterns have been modified to match more general fuzzer cases